exactschedule.go

```diff
@@ -0,0 +1,20 @@
+package cron
+
+import "time"
+
+// ExactSchedule represents a schedule that will only run at the exact time and date provided.
+type ExactSchedule struct {
+	Schedule time.Time
+}
+
+// Next returns the next time this should be run.
+// This rounds so that the next activation time will be on the second.
```

Comment:
This comment doesn't make much sense to me. Has it been copy pasted?

Reply:
Yeah bad commenting. Fixed: https://github.com/NeilGerring/cron/pull/1/commits/3c74ada4d9af62ef6b3a047223a4137c3c894a0b